server/routes/aws/__init__.py

```
@@ -2,11 +2,10 @@
 bp = Blueprint('aws', __name__)
 
-from . import aws_routes, auth, onboarding
-from .cloudwatch_routes import cloudwatch_bp
-from . import cloudwatch_tasks  # noqa: F401 — registers Celery task
+from . import aws_routes, auth, onboarding, securityhub_routes
 
 bp.register_blueprint(aws_routes.aws_bp)
 bp.register_blueprint(auth.auth_bp)
 bp.register_blueprint(onboarding.onboarding_bp)
+bp.register_blueprint(securityhub_routes.securityhub_bp, url_prefix='/securityhub')
 bp.register_blueprint(cloudwatch_bp)
```

Ruff (0.15.14)

[error] Undefined name `cloudwatch_bp`

(F821)